src/client/frugalos.rs

```diff
@@ -179,6 +180,21 @@ impl Client {
         )
     }
 
+    /// Executes `RepairObjectRpc`.
+    pub fn repair_by_ids(
+        &self,
+        bucket_id: BucketId,
+        object_ids: BTreeSet<ObjectId>,
+    ) -> impl Future<Item = (), Error = Error> {
```

Comment:
This function requires neither `segment` nor `DeviceId` because we apply a repair operation to all of `Node`s.

See https://github.com/frugalos/frugalos/pull/40/files#diff-52d7d4c1d9499cddc71c95ec2c87dc1fR81